components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -226,6 +231,8 @@ A warning appears if members are still attached to the policy, because deleting
 However, you can still delete the policy.
 Repeat for each legacy policy.
 
+Alternately, you can run `--skip-policy-migration` on upgrade to start without any policies at all.
```

Comment:
This line should be removed as at this stage of the document, you're past the point of upgrading. You've also done a very thorough job of including this information elsewhere in the document.

Reply:
Erring on the side of redundancy 😂